packages/flutter_tools/test/general.shard/compile_test.dart

```diff
@@ -264,50 +264,6 @@ example:org-dartlang-app:/
       Platform: _kNoColorTerminalPlatform,
     });
 
-    testUsingContext('compile and recompile', () async {
```

Comment:
This test fails with the change, I am assuming it is incorrect

Reply:
How does it fail? What was it trying to test? Should we still have test coverage of whatever it was trying to test?